sling.go

```diff
@@ -269,12 +266,6 @@ func (s *Sling) QueryValues(values url.Values) *Sling {
 	return s
 }
 
-// Allows you to set a tracer to facilitate distributed tracing.
-func (s *Sling) Tracer(tracer Tracer) *Sling {
-	s.tracer = tracer
-	return s
-}
-
```

Comment:
I ended up reverting `Tracer` once I found out that it can be rolled into `Doer`.

Reply:
Sounds good.